benchmark/strong_scaling_incompressible_model_single.jl

```diff
@@ -10,36 +10,41 @@ using Benchmarks
 Logging.global_logger(OceananigansLogger())
 
 MPI.Init()
-comm = MPI.COMM_WORLD
 
+      comm = MPI.COMM_WORLD
 local_rank = MPI.Comm_rank(comm)
-R = MPI.Comm_size(comm)
+         R = MPI.Comm_size(comm)
 
-Nx = parse(Int, ARGS[1])
-Ny = parse(Int, ARGS[2])
-Nz = parse(Int, ARGS[3])
+decomposition = ARGS[1]
+Nx = parse(Int, ARGS[2])
+Ny = parse(Int, ARGS[3])
+Nz = parse(Int, ARGS[4])
+Rx = parse(Int, ARGS[5])
+Ry = parse(Int, ARGS[6])
+Rz = parse(Int, ARGS[7])
 
-@info "Setting up distributed incompressible model with N=($Nx, $Ny, $Nz) grid points on $R rank(s)..."
+@assert Rx * Ry * Rz == R
+
+@info "Setting up distributed incompressible model with N=($Nx, $Ny, $Nz) grid points and ranks=($Rx, $Ry, $Rz) on rank $local_rank..."
```

Comment:
Do we want to say whether it's slab or pencil?  Not even sure if we can but just a thought

Reply:
Ah we can definitely do that!

Reply:
Only if you like